src/lightning_utilities/core/rank_zero.py

```diff
@@ -13,7 +13,10 @@
 
 
 def rank_zero_only(fn: Callable) -> Callable:
-    """Function that can be used as a decorator to enable a function/method being called only on global rank 0."""
+    """Wrap a function to call internal function only in rank zero.
+
+    Function that can be used as a decorator to enable a function/method being called only on global rank 0.
```

Comment:
What was wrong with the previous description?

```suggestion
    """Function that can be used as a decorator to enable a function/method being called only on global rank 0.
```

Reply:
it is not declarative